drivers/display/gd7965.c

```diff
@@ -429,7 +429,7 @@ static struct gd7965_data gd7965_driver = {
 	.config = &gd7965_config
 };
 
-static const struct gd7965_config gd7965_config {
+static const struct gd7965_config gd7965_config = {
```

Comment:
Do we ever build this driver in CI?

Reply:
I guess not

Reply:
It's defined as part of a `build_all` test, https://github.com/zephyrproject-rtos/zephyr/blob/fab291779a4d857a09e2a21701f83b9f07c9c5db/tests/drivers/build_all/sensor/spi.dtsi#L83 but the test is missing the Kconfig options to actually compile it (`CONFIG_DISPLAY`, `CONFIG_GD7965`).

Reply:
should actually not be in sensor/, right?

Reply:
true, but there isn't currently a `build_all/display`